chromadb/api/segment.py

```diff
@@ -101,6 +101,7 @@ def __init__(self, system: System):
         self._settings = system.settings
         self._sysdb = self.require(SysDB)
         self._manager = self.require(SegmentManager)
+        self._quota = self.require(QuotaEnforcer)
```

Comment:
Should we enforce metadata checks on the collection level?

Reply:
Good question, I will ask with the team.